packages/dapi-grpc/clients/platform/v0/python/platform_pb2_grpc.py

```
@@ -541,6 +546,17 @@ def getGroupActionSigners(self, request, context):
         context.set_details('Method not implemented!')
         raise NotImplementedError('Method not implemented!')
 
+    def subscribePlatformEvents(self, request, context):
```

Ruff (0.14.2)

Unused method argument: `request`

(ARG002)